addons/im_livechat/static/src/embed/common/discuss_channel_model_patch.js

```diff
@@ -32,5 +32,8 @@ const discussChannelPatch = {
             inverse: "activeLivechats",
         });
     },
+    get hasMemberList() {
+        return false;
+    },
```

Comment:
We no longer make conditions based on existence of file in a bundle.
This should be a proper condition based on env/state of records instead.

Why do we even want to prevent the member list in embed?